Assets/BossRoom/Scripts/Shared/NetworkDoorState.cs

```diff
@@ -1,12 +1,14 @@
 using MLAPI;
 using MLAPI.NetworkVariable;
-using System.Collections;
 using UnityEngine;
 
-/// <summary>
-/// Network state for a door which can be opened by pressing on a floor switch.
-/// </summary>
-public class NetworkDoorState : NetworkBehaviour
+namespace BossRoom
 {
-    public NetworkVariableBool IsOpen { get; } = new NetworkVariableBool();
+    /// <summary>
+    /// Network state for a door which can be opened by pressing on a floor switch.
+    /// </summary>
+    public class NetworkDoorState : NetworkBehaviour
+    {
+        public NetworkVariableBool IsOpen { get; } = new NetworkVariableBool();
```

Comment:
Ah, sadly this definition of `NetworkVariableBool` is removed in further versions of the netcode. Not an issue at this point, just remarking on the fact that it will have to be repalced with `NetworkVariable<bool>` at a later point

Reply:
same goes for all other non-generic NetworkVariable invocations